src/react/src/components/CookiePreferencesText.jsx

```diff
@@ -3,24 +3,15 @@ import { InfoLink, InfoPaths } from '../util/constants';
 
 const CookiePreferencesText = () => (
     <div>
-        The Open Supply Hub uses cookies to collect and analyze site performance
-        and usage. By clicking the Accept button, you agree to allow us to place
-        cookies and share information with Google Analytics. For more
-        information, please visit our{' '}
+        We use cookies to give you the best experience on Open Supply Hub and
+        understand site performance. By accepting, you consent to our use of
+        cookies and other analytics tools according to our{' '}
```

Comment:
Can you use a regular space here instead of `{' '}` ?

Reply:
+1

Reply:
his is used to put an explicit space in a text block, since leading and trailing spaces are ignored at compile/transformation time when there is another tag.

Reply:
Ok, let's keep it if it works as expected.